src/core/namespace.c

```diff
@@ -204,6 +206,7 @@ static const char * const mount_mode_table[_MOUNT_MODE_MAX] = {
         [READONLY]             = "read-only",
         [READWRITE]            = "read-write",
         [TMPFS]                = "tmpfs",
+        [MOUNT_PATHS]           = "mount-path",
```

Comment:
indentation off

why plural?

Reply:
Like BindPaths, this directive takes a list of multiple image:dst tuples